doc/installation/Developer-Setup.md

```diff
@@ -463,11 +465,11 @@ In the `e2e-test` folder you can run the following commands:
 You need to run cypress while also specifying the urls of the frontend, api and excel export service you are using
 
 ```bash
-npm run cypress -- --config baseUrl=http://localhost:3000 --env API_BASE_URL=http://localhost:8080,EXPORT_SERVICE_BASE_URL=http://localhost:8888
+npm run cypress -- --config baseUrl=http://localhost:3000 --env API_BASE_URL=http://localhost:8080,EXPORT_SERVICE_BASE_URL=http://localhost:8888,ROOT_SECRET=root-secret
```

Comment:
@galethil when you are using .env file, there is no need to use --env flag. The .env file has precedence. There should be mention of two possible runs: One with .env file and the second with --env flag to not cause confusion.

Reply:
@Stezido what is the preferred way of running e2e tests? What should be put to Developer documentation?

Reply:
@m-janos yes true but we have to make sure the executor is in the right folder
@galethil If all e2e-test should be executed and only the end result is important the all-in-one-docker-compose should be used. If the docker-compose is already running of course only the required services and the e2e-tests can be restarted. If I only want to test a specific test I would use the ui with npm run cypress or the script with npm run e2e-test:here. We should definitely document the prefered ways for both scenarios:
1. How to run e2e-tests while I can do other stuff and only want the end result.
2. How to run only a set of tests so the try and error cycle is as fast as possible

Reply:
I will only add the all-in-one docker-compose documentation in this PR. The other changes could be done in  https://github.com/openkfw/TruBudget/issues/769